### ISSUE TITLE
Clicking on project name in the Open dialog makes the project list jump down

### ISSUE THREAD

**Issue by brianharvey** [jmoenig/Snap#2608]

**Comment by jguille2** [comment]
Hi Brian,
I'm testing this (in Chromium-Linux, in current and dev) and I can't reproduce it. I can't shifted  more than one project in no case.

Joan

**Comment by brianharvey** [comment]
No, one project of shift is what I'm talking about.  It should be zero projects shift.
